examples/hello/test/hello-world.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+import {test} from 'babel-tap';
+import initTestium from 'testium-core';
+
+import createDriver from '../../..';
+
+test('Load example page', async t => {
+  const { browser } = await initTestium().then(createDriver);
```

Comment:
@EndangeredMassa This would be the interface the test framework integrations would be working against, e.g. `testium-mocha`:

```bash
# Preparation
npm install --save-dev testium-driver-sync testium-core
```

```coffee
# ~= testium-mocha
injectBrowser = (options) ->
  -> # before hook using mocha's promise support
    initTestium(options)
      .then (testium) ->
        driverType = testium.config.get('driverType', 'sync')
        driverType = require("testium-driver-#{driverType}") if typeof driverType == 'string'
        driverType testium
      .then ({@browser}) => # injected browser

describe 'my feature', ->
  before injectBrowser driverType: 'sync'

  it 'loads a page', ->
    @browser.navigateTo '/'
```